client/clusters/clusters_client.go

```diff
@@ -62,7 +62,7 @@ func (a *Client) DeleteSingleCluster(params *DeleteSingleClusterParams) (*Delete
 		PathPattern:        "/clusters/{name}",
 		ProducesMediaTypes: []string{"application/json"},
 		ConsumesMediaTypes: []string{"application/json"},
-		Schemes:            []string{"http", "https"},
+		Schemes:            []string{"https"},
```

Comment:
Okay, strange, but this is what I find:

I have the same swagger version as noted in the tls PR.  When I set my api/swagger.yaml to include both http and https as mentioned above in elmiko's comment, I get code that looks like the generated code in tls :)  So I wonder if it's possible that the generated code in the tls PR was created with that definition in api/swagger.yaml, and then somehow api/swagger.yaml was reverted.

At any rate, I think it's the correct thing to have and I will submit a change momentarily ...